src/Header/AppHeader.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import { ButtonV2 } from '../ButtonV2';
+import { LogoutIcon, SwitchAppIconColour } from '../Icons';
+import {
+  ItemWrapper,
+  StyledButtonTitle,
+  StyledNewBadge,
+  StyledText,
+  SwitchAppWrapper,
+} from './ProductSwitch/ProductSwitch.styles'; // TODO: ProductSwitcher still needs some work, will do another PR
+import { List } from '../List';
+import { Popover } from '../Popover';
+import { EdozoLogo } from '../EdozoLogo';
+import { Menu } from './Menu';
+import { Context, Header } from './Header';
+
+export interface ProductInfo {
+  appName: string;
+  description: string;
+  productionUrl: string;
+  stageUrl: string;
+}
+
+const defaultProducts: ProductInfo[] = [
+  {
+    appName: '',
+    description: '',
+    productionUrl: 'https://dashboard.edozo.com/',
+    stageUrl: 'https://dashboard.edozo.co/',
+  },
```

Comment:
Just noticed that in the header story, this adds an extra list item in the product switch.

Reply:
ah yeah, i'll remove that one - I had is to we had that option in the story controls